lighthouse-core/lib/i18n/locales/ar-XB.json

```diff
@@ -779,39 +779,21 @@
   "lighthouse-core/audits/metrics/estimated-input-latency.js | description": {
     "message": "‏‮Estimated‬‏ ‏‮Input‬‏ ‏‮Latency‬‏ ‏‮is‬‏ ‏‮an‬‏ ‏‮estimate‬‏ ‏‮of‬‏ ‏‮how‬‏ ‏‮long‬‏ ‏‮your‬‏ ‏‮app‬‏ ‏‮takes‬‏ ‏‮to‬‏ ‏‮respond‬‏ ‏‮to‬‏ ‏‮user‬‏ ‏‮input‬‏, ‏‮in‬‏ ‏‮milliseconds‬‏, ‏‮during‬‏ ‏‮the‬‏ ‏‮busiest‬‏ 5‏‮s‬‏ ‏‮window‬‏ ‏‮of‬‏ ‏‮page‬‏ ‏‮load‬‏. ‏‮If‬‏ ‏‮your‬‏ ‏‮latency‬‏ ‏‮is‬‏ ‏‮higher‬‏ ‏‮than‬‏ 50 ‏‮ms‬‏, ‏‮users‬‏ ‏‮may‬‏ ‏‮perceive‬‏ ‏‮your‬‏ ‏‮app‬‏ ‏‮as‬‏ ‏‮laggy‬‏. [‏‮Learn‬‏ ‏‮more‬‏](https://web.dev/estimated-input-latency)."
   },
-  "lighthouse-core/audits/metrics/estimated-input-latency.js | title": {
-    "message": "‏‮Estimated‬‏ ‏‮Input‬‏ ‏‮Latency‬‏"
```

Comment:
this is a massive downside to the approach of removing obsolete strings immediately, @brendankenny any thoughts here?

Reply:
> this is a massive downside to the approach of removing obsolete strings immediately, @brendankenny any thoughts here?

the `locales` id changed (from e.g. `lighthouse-core/audits/metrics/estimated-input-latency.js | title` to `lighthouse-core/lib/i18n/i18n.js | estimatedInputLatency`) so we'd fall back to the english string even if we didn't remove them, wouldn't we? We'd have the translations but we couldn't use them, so it does seem better to prune in that case.

We could identify identical string/description moving and move the strings in each locale file, but I think that should end up happening anyways the next time we get a translation dump. I believe tc does id by hashing the string+description (+optional meaning?), so in theory we should get new translations immediately. I think that means we should be careful not to reword any strings or their descriptions if we want that to happen :)

I'm not sure exactly how that will work in practice or if I have all the details right. cc @exterkamp

Reply:
Re: Timing Budgets Setup
I’m leaning towards adding it as a "timing-budget" audit that is part of the budgets group. (And the existing “performance budget” audit becomes “resource-budget”).

An alternative to this would be to keep everything in one audit and to refactor the table to be more generic with headers like:
Description / Measurement / Budget / Over Budget

However, I’m not wild about the JSON that produces; it would be mixing items with different units (e.g. KB vs ms) in `details.items`. Using a non-standard structure for details.items could side step this... but would be non-standard so I'm not wild about that either.

Reply:
> we'd fall back to the english string even if we didn't remove them, wouldn't we?

oh, ya ya, my b. we don't have locale-versioned code yet :) sorry for the noise

Reply:
> I'm not sure exactly how that will work in practice or if I have all the details right. cc @exterkamp

This is exactly right _in theory_. This will be the first large scale test of moving strings. But in theory, this should be a no-op for tc/ since the text is verbatim. Should be able to land this, then re-import  immediately. However, since this will be in the middle of another half complete translation, it would add a lot of English strings to stack packs. So I guess it's a decision whether we want to do this immediately or wait until all 6.0.0 i18n is ready. Just makes more `i18n(import)` PRs.